frontend/src/component/admin/license/License.tsx

```diff
@@ -0,0 +1,140 @@
+    display: 'table-cell',
+    fontWeight: theme.fontWeight.bold,
+    paddingTop: theme.spacing(2),
+}));
+
+const StyledPropertyDetails = styled('p')(({ theme }) => ({
+    display: 'table-cell',
+    paddingTop: theme.spacing(2),
+    paddingLeft: theme.spacing(4),
+}));
+
+
+
+
+
+export const License = () => {
+    const { setToastData, setToastApiError } = useToast();
+    const { license, error, refetchLicense } = useLicense();
+    const { reCheckLicense } = useLicenseCheck();
+    const { loading } = useUiConfig();
+    const { locationSettings } = useLocationSettings();
+    const [token, setToken] = useState('');
+    const { updateLicenseKey } = useUpdateLicenseKeyApi();
+
+    const updateToken = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setToken(event.target.value.trim());
+    };
+
+    if (loading || !license) {
+        return null;
+    }
+
+    const onSubmit = async (event: React.SyntheticEvent) => {
+        event.preventDefault();
+
+        try {
+            await updateLicenseKey(token);
+            setToastData({
+                title: 'License key updated',
+                type: 'success',
+            });
+            refetchLicense();
+            reCheckLicense ();
+        } catch (error: unknown) {
+            setToastApiError(formatUnknownError(error));
+        }
+    };
+
+    return (
+        <PageContent header={<PageHeader title='Unleash Enterprise License' />}>
+            <StyledBox>
+                {license.token ? <div>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Customer</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {license?.customer}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Plan</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {license?.plan}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Seats</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {license?.seats}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Expire at</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {formatDateYMD(license.expireAt, locationSettings.locale)}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                </div>: <p>You do not have a registered Unleash Enterprise License.</p>}
```

Comment:
Using a `ConditionallyRender` would be more consistent with the rest of the codebase instead of the ternary.